test/python/test_pack.py

```diff
@@ -256,3 +264,27 @@ def test_packing_without_git(tmpdir):
         project_path,
     ]
     process = subprocess.run(cmd, cwd=tmpdir)
+
+
+def test_packing_with_wrong_filemodes(tmpdir):
```

Comment:
I think this test should be moved to separate patch/commit

It's not about "new package flow"

Reply:
See #102